Tests/StoreKitUnitTests/StoreProductTests.swift

```diff
@@ -299,6 +299,33 @@ class StoreProductTests: StoreKitConfigTestCase {
         expect(storeProduct.currencyCode) == "USD"
     }
 
+    @available(iOS 16.0, tvOS 16.0, macOS 13.0, watchOS 9.0, *)
+    func testSk2PriceFormatterGetsLocaleFromStoreKit() async throws {
```

Comment:
This is equivalent to the previous test `testSk2PriceFormatterReactsToStorefrontChanges`, except it doesn't need to manually change the locale in the `priceFormatter`